tasks/install.js

```diff
@@ -25,10 +25,11 @@ module.exports = function(grunt) {
 
   if (dbUrl) {
     dbUrl = '--db-url=' + dbUrl;
+    args.push(dbUrl);
   }
 
   grunt.config(['drush', 'install'], {
-    args: args.concat(['site-install', '-yv', profile, dbUrl]),
+    args: args.concat(['site-install', '-yv', profile]),
```

Comment:
@jhedstrom it seems like this means the --db-url flag would be serialized into the command before site-install, e.g., `drush --db-url=mysql://... site-install -yv standard`. Would that work?

Reply:
@grayside in testing drush seemed not to care (I had the same thought).